source/guides/stacks/mers.rst

```diff
@@ -208,45 +249,93 @@ This example shows how to perform transcoding with :command:`FFmpeg`.
 
       cp </path/to/video> ~/ffmpeg/input
 
-#. Run the *clearlinux/stack-mers* docker image, allowing shared access to the
-   workspace on the host:
-
+#. Run the *sysstacks/mers-clearlinux* docker image, allowing shared access to
+   the workspace on the host:
 
    .. code:: bash
 
       docker run -it \
-      -v ~/ffmpeg:/home/mers-user:ro \
-      clearlinux/stacks-mers:latest
+      --volume ~/ffmpeg:/home/mers-user:ro \
+      --device=/dev/dri \
+      --env QSV_DEVICE=/dev/dri/renderD128 \
+      sysstacks/mers-clearlinux:latest
+
+   .. note::
+
+      The :command:`--device` parameter and the **GSV_DEVICE** environment
+      variable allow shared access to the GPU on the host system. The values
+      needed may be different depending on host's graphics configuration.      
 
    After running the :command:`docker run` command, you enter a bash shell
    inside the container. 
 
-#. From the container shell, you can run :command:`FFmpeg` against the videos
-   in :file:`/home/mers-user/input` as you would normally outside of |MERS|.
+#. From the container shell, you can run :command:`FFmpeg` and
```

Comment:
Per the CL style guide, I'd not use the :command: role here for the FFmpeg and Gstreamer (also no italic or bold) since it is referring to a tool name. Also applies to the 'for more info' directly below (no italic).